test/testutil/mocks/storage/ipfs/ipfs_datastore_mock.hpp

```diff
@@ -18,6 +18,9 @@ namespace fc::storage::ipfs {
     MOCK_METHOD2(set, outcome::result<void>(const CID &key, Value value));
     MOCK_CONST_METHOD1(get, outcome::result<Value>(const CID &key));
     MOCK_METHOD1(remove, outcome::result<void>(const CID &key));
+    IpldPtr shared() override {
```

Comment:
no no no 

We don't talk here about actual implementation.
We just define here mocked method signature.

```
MOCK_METHOD0(shared, IpldPtr());
```